test/e2e/metamask-ui.spec.js

```diff
@@ -449,8 +450,9 @@ describe('MetaMask', function () {
     });
 
     it('customizes gas', async function () {
+      await driver.delay(1000);
       await driver.clickElement({ text: 'Edit', tag: 'button' });
-      await driver.delay(largeDelayMs);
+      await driver.delay(1000);
```

Comment:
NIT: There are 7 occurrences of hardcoding `1000` in this file before this change, now there are 8. ;) Probably we should use this opportunity to make this a constant

```suggestion
      await driver.delay(veryLargeDelayMs);
```

Reply:
done here https://github.com/MetaMask/metamask-extension/pull/13788/commits/86826e7812064f1be75d04d852659b13cd212106